exotations/exotica_core_task_maps/test/test_maps.cpp

```diff
@@ -1159,11 +1159,37 @@ TEST(ExoticaTaskMaps, testManipulability)
     }
 }
 
+TEST(ExoticaTaskMaps, testAvoidLookAt)
+{
+    try
+    {
+        TEST_COUT << "AvoidLookAt Test";
+        Initializer map("exotica/AvoidLookAt", {{"Name", std::string("MyTask")},
+						{"EndEffector", std::vector<Initializer>(
+											 {Initializer("EffPoint", {{"Link", std::string("EffPoint")},
+														   {"Base", std::string("base")},
+														   {"BaseOffset", std::string("0 0 1")}}),
+											  Initializer("EffPoint", {{"Link", std::string("Obj")},
```

Comment:
Should this `Initializer("EffPoint",` be this `Initializer("Frame",` ?

Reply:
Ahh yes, will update